### PR TITLE
fix: Range request

### PR DESCRIPTION
**Pull request by wayfarer3130**
Adds fixes for range request

### CODE REVIEW

File: packages/s3-deploy/test [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{
```
Comment (sedghi) [inline packages/s3-deploy/test:1]: do you need this?

File: packages/static-wado-webserver/lib/controllers/server/byteRangeRequest.mjs [hunk 2]
```diff
@@ -17,44 +25,55 @@ export default function byteRangeRequest(options) {
...
     const rawdata = await fs.promises.readFile(path);
     const { length } = rawdata;
     bytes[0] ||= 0;
-    bytes[1] ||= length;
-    bytes[1] = Math.min(bytes[1], length);
-    const data = rawdata.slice(bytes[0], bytes[1]);
-    res.setHeader("content-type", "multipart/related");
-    res.setHeader("Access-Control-Allow-Headers", "Content-Range");
-    res.setHeader("Content-Range", `${bytes[0]}-${bytes[1]}/${length}`);
+    bytes[1] ||= length-1;
```
Comment (sedghi) [inline packages/static-wado-webserver/lib/controllers/server/byteRangeRequest.mjs:53]: shouldn't this really be the length?

File: package.json [hunk 3]
```diff
@@ -62,5 +62,9 @@
     "lint:fix": "lerna run lint:fix --parallel --stream",
     "postinstall": "git config core.hooksPath '.husky' && echo 'git hooks configured'",
     "reset:hooks": "git config core.hooksPath '.git' && echo 'git hooks was reset'"
+  },
+
+  "resolutions": {
```
Comment (sedghi) [inline package.json:67]: there is already a resolutions in the top

**Review by sedghi — COMMENTED** [review]
see my comments

File: packages/static-wado-webserver/lib/controllers/server/byteRangeRequest.mjs (hunk 2 above)
Comment (wayfarer3130) [reply packages/static-wado-webserver/lib/controllers/server/byteRangeRequest.mjs:53]: Absolutely not - if you set it that way it kills the video client.  The range is inclusive at both ends.

**Pull request merged**